src/core/apolloClient/cache.ts

```diff
@@ -67,7 +67,7 @@ export const cache = new InMemoryCache({
           keyArgs: ['invoiceId'],
         },
         customerInvoices: {
```

Comment:
I think you forgot to add that for the other search queries no ?

Reply:
I had no issue with others resources

Basically, I mostly want to cache results no matter the search result.

`customerInvoices` case was a bit different: There is 2 query containing ont data that is stored in the same object, but can contains a different value: metadata.totalCount.

The first one is used for the load on scroll, and needs to change depending on the number of results prompted on the screen.
The second one is used to get the total number of invoices (draft in this case), and this value should not change depending on the `searchTerm`.

Another approach could have been to store the initial `totalCount` value in a local state but I preferred the cache approach, avoiding "complexity" in the component itself.

Happy to discuss about it !

Reply:
Ok got it
I think is not the best option then as then the caching is ont really happening for this query (each time you have a new search, the items are stored with a new key, apollo does not get the ones already fetched)

So the 2 options are :
- The one you mentionned
- Having a separated query with no search that gets the totalCount only